app/src/main/res/layout/header_logo.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/logo"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:foreground="?selectableItemBackground"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toTopOf="parent">
+
+    <ImageView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:layout_marginTop="@dimen/margin_large"
+        android:layout_marginBottom="@dimen/margin_standard"
+        android:padding="@dimen/margin_standard"
+        android:src="@drawable/ic_logo"
+        tools:ignore="ContentDescription" />
```

Comment:
I think this should be `android:importantForAccessibility="no"` instead.
That icon is not important for a screen readers or other accessibility tools. `ignore` looks more like silencing warnings, and though ChessClock is unfortunately not to accessible we can probably still differentiate between "not important" and "we don't have resources to figure this out" cases.